Lps 59066

shuyangzhou opened the pull request.

shuyangzhou committed:
LPS-59066 jConnect driver put in prefix to all exception messages, change ExpectedLog assertion to do contains.

shuyangzhou committed:
LPS-59066 Sybase does not allow empty string value for VARCHAR type column, on see an empty string insert or update for VARCHAR type column, the value will be changed to ' ' (space character) on the server side. In the old jtds.jar, on reading a VARCHAR type column, a single space value is translated to empty string (see http://ehc.ac/p/jtds/discussion/104389/thread/93a9aa08/ for more info). But jConnect does no such thing, it directly returns the single space char string, therefore causing a lot of troubles. In this commit, we translate empty string to null at hibernate level, to avoid the confusion when we read it back.

shuyangzhou committed:
LPS-59066 jConnect driver names itself "Adaptive Server Enterprise"

shuyangzhou committed:
LPS-59066 Remove jtds.jar

shuyangzhou committed:
LPS-59066 ant build-lib-versions

shuyangzhou committed:
LPS-59066 Projects

shuyangzhou committed:
LPS-59066 Remove references from build scripts

shuyangzhou committed:
LPS-59066 Switch to use jconn4.jar driver

shuyangzhou committed:
LPS-59066 Patch SybaseASE157Dialect to map jdbc timestamp type to sybase bigdatetime in order to preserve accurate milliseconds info

shuyangzhou committed:
LPS-59066 Change SybaseDB to map timestamp to bigdatetime

shuyangzhou committed:
LPS-59066 SQLDateTest to ensure all DBs can preserve milliseconds info correctly

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 1 hour 7 minutes 50 seconds</p><h4>Job Results:</h4><p>20 Jobs Passed.<br />0 Jobs Failed.</p><h1>See the following links for more information.</h1><ul><li><h4><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/1393/">test-portal-acceptance-pullrequest(master)</a></h4><pre>Completed with the status of FAILURE.</pre></li><li><h4><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2736/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/0</a></h4></li><li><h4><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2714/">test-portal-acceptance-pullrequest-batch(master)/integration-db2</a></h4></li><li><h4><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2670/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic</a></h4></li><li><h4><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9621/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql</a></h4></li><li><h4><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9694/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle</a></h4></li><li><h4><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9605/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql</a></h4></li><li><h4><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2708/">test-portal-acceptance-pullrequest-batch(master)/integration-sybase</a></h4></li><li><h4><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2734/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2</a></h4></li><li><h4><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2710/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic</a></h4></li><li><h4><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9610/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql</a></h4></li><li><h4><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2718/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle</a></h4></li><li><h4><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9567/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql</a></h4></li><li><h4><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1177/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase</a></h4></li><li><h4><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9708/">test-portal-acceptance-pullrequest-batch(master)/modules-unit</a></h4></li><li><h4><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9694/">test-portal-acceptance-pullrequest-batch(master)/plugins</a></h4></li><li><h4><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1149/">test-portal-acceptance-pullrequest-batch(master)/portal-web</a></h4></li><li><h4><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9645/">test-portal-acceptance-pullrequest-batch(master)/service-builder</a></h4></li><li><h4><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1152/">test-portal-acceptance-pullrequest-batch(master)/tck</a></h4></li><li><h4><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1165/">test-portal-acceptance-pullrequest-batch(master)/unit</a></h4></li><li><h4><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/219/">test-portal-acceptance-pullrequest-source(master)</a></h4></li></ul>

shuyangzhou commented:
@michaelhashimoto

The build completed successfully, but jenkins failed at the end for publishing results, it ran out of memory for loading some scripts from ant build xml. Please check on this, I saw this happened to a lot of pulls.

```
     [echo] Build 'test-portal-acceptance-pullrequest-batch(master)/unit' started at https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1165/.
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2736/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2714/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2670/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9621/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9694/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9605/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2708/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2734/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2710/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9610/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2718/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9567/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1177/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9708/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9694/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1149/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9645/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1152/. SUCCESS
     [echo] 'test-portal-acceptance-pullrequest-batch(master)' completed at https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1165/. SUCCESS
   [script] Sep 29, 2015 4:46:04 PM org.apache.bsf.BSFManager loadScriptingEngine
   [script] SEVERE: Exception :
   [script] java.lang.OutOfMemoryError
   [script] 	at java.util.zip.Inflater.inflateBytes(Native Method)
   [script] 	at java.util.zip.Inflater.inflate(Inflater.java:259)
   [script] 	at java.util.zip.InflaterInputStream.read(InflaterInputStream.java:152)
   [script] 	at org.apache.tools.ant.AntClassLoader.getClassFromStream(AntClassLoader.java:1303)
   [script] 	at org.apache.tools.ant.AntClassLoader.findClassInComponents(AntClassLoader.java:1362)
   [script] 	at org.apache.tools.ant.AntClassLoader.findClass(AntClassLoader.java:1323)
   [script] 	at org.apache.tools.ant.AntClassLoader.loadClass(AntClassLoader.java:1076)
   [script] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
   [script] 	at bsh.Interpreter.<init>(Unknown Source)
   [script] 	at bsh.Interpreter.<init>(Unknown Source)
   [script] 	at bsh.Interpreter.<init>(Unknown Source)
   [script] 	at bsh.util.BeanShellBSFEngine.initialize(Unknown Source)
   [script] 	at org.apache.bsf.BSFManager$8.run(BSFManager.java:703)
   [script] 	at java.security.AccessController.doPrivileged(Native Method)
   [script] 	at org.apache.bsf.BSFManager.loadScriptingEngine(BSFManager.java:701)
   [script] 	at org.apache.bsf.BSFManager.exec(BSFManager.java:485)
   [script] 	at org.apache.tools.ant.util.optional.ScriptRunner.executeScript(ScriptRunner.java:100)
   [script] 	at org.apache.tools.ant.taskdefs.optional.Script.execute(Script.java:52)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
   [script] 	at net.sf.antcontrib.logic.ForTask.doSequentialIteration(ForTask.java:259)
   [script] 	at net.sf.antcontrib.logic.ForTask.doToken(ForTask.java:268)
   [script] 	at net.sf.antcontrib.logic.ForTask.doTheTasks(ForTask.java:299)
   [script] 	at net.sf.antcontrib.logic.ForTask.execute(ForTask.java:244)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at net.sf.antcontrib.logic.IfTask.execute(IfTask.java:197)
   [script] 	at sun.reflect.GeneratedMethodAccessor52.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.TaskAdapter.execute(TaskAdapter.java:154)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
   [script] 	at net.sf.antcontrib.logic.ForTask.doSequentialIteration(ForTask.java:259)
   [script] 	at net.sf.antcontrib.logic.ForTask.doToken(ForTask.java:268)
   [script] 	at net.sf.antcontrib.logic.ForTask.doTheTasks(ForTask.java:324)
   [script] 	at net.sf.antcontrib.logic.ForTask.execute(ForTask.java:244)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at net.sf.antcontrib.logic.IfTask.execute(IfTask.java:197)
   [script] 	at sun.reflect.GeneratedMethodAccessor52.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.TaskAdapter.execute(TaskAdapter.java:154)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
   [script] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
   [script] 	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
   [script] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   [script] 	at java.lang.reflect.Method.invoke(Method.java:606)
   [script] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
   [script] 	at org.apache.tools.ant.Task.perform(Task.java:348)
   [script] 	at org.apache.tools.ant.Target.execute(Target.java:435)
   [script] 	at org.apache.tools.ant.Target.performTasks(Target.java:456)
   [script] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1393)
   [script] 	at org.apache.tools.ant.Project.executeTarget(Project.java:1364)
   [script] 	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
   [script] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1248)
   [script] 	at org.apache.tools.ant.Main.runBuild(Main.java:851)
   [script] 	at org.apache.tools.ant.Main.startAnt(Main.java:235)
   [script] 	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:280)
   [script] 	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:109)
   [script] 
Caught an exception while logging the end of the build.  Exception was:
java.lang.OutOfMemoryError
	at java.util.zip.Inflater.inflateBytes(Native Method)
	at java.util.zip.Inflater.inflate(Inflater.java:259)
	at java.util.zip.InflaterInputStream.read(InflaterInputStream.java:152)
	at sun.misc.Resource.getBytes(Resource.java:124)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:444)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at org.apache.tools.ant.DefaultLogger.formatTime(DefaultLogger.java:330)
	at org.apache.tools.ant.DefaultLogger.buildFinished(DefaultLogger.java:177)
	at org.apache.tools.ant.Project.fireBuildFinished(Project.java:2087)
	at org.apache.tools.ant.Main.runBuild(Main.java:872)
	at org.apache.tools.ant.Main.startAnt(Main.java:235)
	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:280)
	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:109)
There has been an error prior to that:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-portal-acceptance-pullrequest.xml:302: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:4014: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2218: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2250: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2254: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2180: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:543: java.lang.OutOfMemoryError
	at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:568)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:399)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.Target.execute(Target.java:435)
	at org.apache.tools.ant.Target.performTasks(Target.java:456)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1393)
	at org.apache.tools.ant.Project.executeTarget(Project.java:1364)
	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1248)
	at org.apache.tools.ant.Main.runBuild(Main.java:851)
	at org.apache.tools.ant.Main.startAnt(Main.java:235)
	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:280)
	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:109)
Caused by: /opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:4014: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2218: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2250: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2254: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2180: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:543: java.lang.OutOfMemoryError
	at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:568)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:399)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at net.sf.antcontrib.logic.IfTask.execute(IfTask.java:197)
	at sun.reflect.GeneratedMethodAccessor52.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.TaskAdapter.execute(TaskAdapter.java:154)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
	... 16 more
Caused by: /opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2218: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2250: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2254: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2180: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:543: java.lang.OutOfMemoryError
	at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:568)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:399)
	at net.sf.antcontrib.logic.ForTask.doSequentialIteration(ForTask.java:259)
	at net.sf.antcontrib.logic.ForTask.doToken(ForTask.java:268)
	at net.sf.antcontrib.logic.ForTask.doTheTasks(ForTask.java:324)
	at net.sf.antcontrib.logic.ForTask.execute(ForTask.java:244)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
	... 43 more
Caused by: /opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2250: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2254: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2180: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:543: java.lang.OutOfMemoryError
	at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:568)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:399)
	at net.sf.antcontrib.logic.ForTask.doSequentialIteration(ForTask.java:259)
	at net.sf.antcontrib.logic.ForTask.doToken(ForTask.java:268)
	at net.sf.antcontrib.logic.ForTask.doTheTasks(ForTask.java:299)
	at net.sf.antcontrib.logic.ForTask.execute(ForTask.java:244)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at net.sf.antcontrib.logic.IfTask.execute(IfTask.java:197)
	at sun.reflect.GeneratedMethodAccessor52.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.TaskAdapter.execute(TaskAdapter.java:154)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
	... 61 more
Caused by: /opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2254: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2180: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:543: java.lang.OutOfMemoryError
	at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:568)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:399)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
	... 92 more
Caused by: /opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:2180: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:543: java.lang.OutOfMemoryError
	at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:568)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:399)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
	... 106 more
Caused by: /opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:543: java.lang.OutOfMemoryError
	at org.apache.tools.ant.util.optional.ScriptRunner.getBuildException(ScriptRunner.java:147)
	at org.apache.tools.ant.util.optional.ScriptRunner.executeScript(ScriptRunner.java:105)
	at org.apache.tools.ant.taskdefs.optional.Script.execute(Script.java:52)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.Sequential.execute(Sequential.java:68)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
	... 120 more
Caused by: java.lang.OutOfMemoryError
	at java.util.zip.Inflater.inflateBytes(Native Method)
	at java.util.zip.Inflater.inflate(Inflater.java:259)
	at java.util.zip.InflaterInputStream.read(InflaterInputStream.java:152)
	at org.apache.tools.ant.AntClassLoader.getClassFromStream(AntClassLoader.java:1303)
	at org.apache.tools.ant.AntClassLoader.findClassInComponents(AntClassLoader.java:1362)
	at org.apache.tools.ant.AntClassLoader.findClass(AntClassLoader.java:1323)
	at org.apache.tools.ant.AntClassLoader.loadClass(AntClassLoader.java:1076)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at bsh.Interpreter.<init>(Unknown Source)
	at bsh.Interpreter.<init>(Unknown Source)
	at bsh.Interpreter.<init>(Unknown Source)
	at bsh.util.BeanShellBSFEngine.initialize(Unknown Source)
	at org.apache.bsf.BSFManager$8.run(BSFManager.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.bsf.BSFManager.loadScriptingEngine(BSFManager.java:701)
	at org.apache.bsf.BSFManager.exec(BSFManager.java:485)
	at org.apache.tools.ant.util.optional.ScriptRunner.executeScript(ScriptRunner.java:100)
	... 135 more
Build step 'Execute shell' marked build as failure
[PostBuildScript] - Execution post build scripts.
[liferay-jenkins-ee] $ /bin/bash /tmp/hudson7129355652493945254.sh
Update the workspace repository for post-build steps.
No rebase in progress?
Switched to branch 'master-temp-773842585'
Switched to branch 'master'
Deleted branch master-temp-773842585 (was a5044f5).
HEAD is now at a5044f5 LRQA-17553 Regen
Rebase the workspace repository if needed for post-build steps.
Exit shell.
[liferay-jenkins-ee] $ /bin/bash -e -x /tmp/hudson3172612209231095090.sh
+ echo 'Execute: test-portal-acceptance-pullrequest.sh stop-current-job'
Execute: test-portal-acceptance-pullrequest.sh stop-current-job
+ commands/build-test-portal-acceptance-pullrequest.sh stop-current-job
Execute: ant -f commands/build-test-portal-acceptance-pullrequest.xml stop-current-job
Buildfile: /opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-portal-acceptance-pullrequest.xml

stop-current-job:

post-github-status-failure:

     [exec]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
     [exec]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
     [exec]                                  Dload  Upload   Total   Spent    Left  Speed
     [exec] 
     [exec]                                  Dload  Upload   Total   Spent    Left  Speed
     [exec] 
     [exec] {
     [exec]   "url": "https://api.github.com/repos/brianchandotcom/liferay-portal/statuses/f915561c918ade2f68638654fe8c4360e9f9f12a",
     [exec]   "id": 320684162,
     [exec]   "state": "failure",
     [exec]   "description": "Some tests FAILED!",
     [exec]   "target_url": "https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1393/jenkins-report.html",
     [exec]   "context": "default",
     [exec]   "created_at": "2015-09-29T23:48:19Z",
     [exec]   "updated_at": "2015-09-29T23:48:19Z",
     [exec]   "creator": {
     [exec]     "login": "liferay-continuous-integration",
     [exec]     "id": 4502793,
     [exec]     "avatar_url": "https://avatars.githubusercontent.com/u/4502793?v=3",
     [exec]     "gravatar_id": "",
     [exec]     "url": "https://api.github.com/users/liferay-continuous-integration",
     [exec]     "html_url": "https://github.com/liferay-continuous-integration",
     [exec]     "followers_url": "https://api.github.com/users/liferay-continuous-integration/followers",
     [exec]     "following_url": "https://api.github.com/users/liferay-continuous-integration/following{/other_user}",
     [exec]     "gists_url": "https://api.github.com/users/liferay-continuous-integration/gists{/gist_id}",
     [exec]     "starred_url": "https://api.github.com/users/liferay-continuous-integration/starred{/owner}{/repo}",
     [exec]     "subscriptions_url": "https://api.github.com/users/liferay-continuous-integration/subscriptions",
     [exec]     "organizations_url": "https://api.github.com/users/liferay-continuous-integration/orgs",
     [exec]     "repos_url": "https://api.github.com/users/liferay-continuous-integration/repos",
     [exec]     "events_url": "https://api.github.com/users/liferay-continuous-integration/events{/privacy}",
     [exec]     "received_events_url": "https://api.github.com/users/liferay-continuous-integration/received_events",
     [exec]     "type": "User",
     [exec]     "site_admin": false
     [exec]   }
     [exec] }
     [exec]   0     0    0     0    0   191      0    737 --:--:-- --:--:-- --:--:--   737111  1706  100  1706    0   191   3890    435 --:--:-- --:--:-- --:--:--  8463
     [exec]   0     0    0     0    0   191      0    737 --:--:-- --:--:-- --:--:--   737111  1706  100  1706    0   191   3890    435 --:--:-- --:--:-- --:--:--  8463
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20150929164818870

post-github-comment:
     [exec]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
     [exec]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
     [exec]                                  Dload  Upload   Total   Spent    Left  Speed
     [exec] 
     [exec]                                  Dload  Upload   Total   Spent    Left  Speed
     [exec] 
     [exec] {
     [exec]   "url": "https://api.github.com/repos/brianchandotcom/liferay-portal/issues/comments/144222701",
     [exec]   "html_url": "https://github.com/brianchandotcom/liferay-portal/pull/30639#issuecomment-144222701",
     [exec]   "issue_url": "https://api.github.com/repos/brianchandotcom/liferay-portal/issues/30639",
     [exec]   "id": 144222701,
     [exec]   "user": {
     [exec]     "login": "liferay-continuous-integration",
     [exec]     "id": 4502793,
     [exec]     "avatar_url": "https://avatars.githubusercontent.com/u/4502793?v=3",
     [exec]     "gravatar_id": "",
     [exec]     "url": "https://api.github.com/users/liferay-continuous-integration",
     [exec]     "html_url": "https://github.com/liferay-continuous-integration",
     [exec]     "followers_url": "https://api.github.com/users/liferay-continuous-integration/followers",
     [exec]     "following_url": "https://api.github.com/users/liferay-continuous-integration/following{/other_user}",
     [exec]     "gists_url": "https://api.github.com/users/liferay-continuous-integration/gists{/gist_id}",
     [exec]     "starred_url": "https://api.github.com/users/liferay-continuous-integration/starred{/owner}{/repo}",
     [exec]     "subscriptions_url": "https://api.github.com/users/liferay-continuous-integration/subscriptions",
     [exec]     "organizations_url": "https://api.github.com/users/liferay-continuous-integration/orgs",
     [exec]     "repos_url": "https://api.github.com/users/liferay-continuous-integration/repos",
     [exec]     "events_url": "https://api.github.com/users/liferay-continuous-integration/events{/privacy}",
     [exec]     "received_events_url": "https://api.github.com/users/liferay-continuous-integration/received_events",
     [exec]     "type": "User",
     [exec]     "site_admin": false
     [exec]   },
     [exec]   "created_at": "2015-09-29T23:48:54Z",
     [exec]   "updated_at": "2015-09-29T23:48:54Z",
     [exec]   "body": "<h1>Some tests FAILED!</h1><p>Build Time: 1 hour 7 minutes 50 seconds</p><h4>Job Results:</h4><p>20 Jobs Passed.<br />0 Jobs Failed.</p><h1>See the following links for more information.</h1><ul><li><h4><a href=\"https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/1393/\">test-portal-acceptance-pullrequest(master)</a></h4><pre>Completed with the status of FAILURE.</pre></li><li><h4><a href=\"https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2736/\">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/0</a></h4></li><li><h4><a href=\"https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2714/\">test-portal-acceptance-pullrequest-batch(master)/integration-db2</a></h4></li><li><h4><a href=\"https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2670/\">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic</a></h4></li><li><h4><a href=\"https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9621/\">test-portal-acceptance-pullrequest-batch(master)/integration-mysql</a></h4></li><li><h4><a href=\"https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9694/\">test-portal-acceptance-pullrequest-batch(master)/integration-oracle</a></h4></li><li><h4><a href=\"https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9605/\">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql</a></h4></li><li><h4><a href=\"https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2708/\">test-portal-acceptance-pullrequest-batch(master)/integration-sybase</a></h4></li><li><h4><a href=\"https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2734/\">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2</a></h4></li><li><h4><a href=\"https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2710/\">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic</a></h4></li><li><h4><a href=\"https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9610/\">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql</a></h4></li><li><h4><a href=\"https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2718/\">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle</a></h4></li><li><h4><a href=\"https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9567/\">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql</a></h4></li><li><h4><a href=\"https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1177/\">test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase</a></h4></li><li><h4><a href=\"https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9708/\">test-portal-acceptance-pullrequest-batch(master)/modules-unit</a></h4></li><li><h4><a href=\"https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9694/\">test-portal-acceptance-pullrequest-batch(master)/plugins</a></h4></li><li><h4><a href=\"https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1149/\">test-portal-acceptance-pullrequest-batch(master)/portal-web</a></h4></li><li><h4><a href=\"https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/9645/\">test-portal-acceptance-pullrequest-batch(master)/service-builder</a></h4></li><li><h4><a href=\"https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1152/\">test-portal-acceptance-pullrequest-batch(master)/tck</a></h4></li><li><h4><a href=\"https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/1165/\">test-portal-acceptance-pullrequest-batch(master)/unit</a></h4></li><li><h4><a href=\"https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/219/\">test-portal-acceptance-pullrequest-source(master)</a></h4></li></ul>"
     [exec] }
     [exec]   0  4196    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100 10040  100  5844  102  4196  11252   8079 --:--:-- --:--:-- --:--:-- 22305
     [exec]   0  4196    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100 10040  100  5844  102  4196  11252   8079 --:--:-- --:--:-- --:--:-- 22305
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20150929164854570

BUILD SUCCESSFUL
Total time: 2 minutes 49 seconds

+ echo 'Exit shell.'
Exit shell.
[EnvInject] - Injecting environment variables from a build step.
[EnvInject] - Injecting as environment variables the properties file path '/mnt/mfs-ssd1-10.0.10/jenkins/test-1-2/opt/java/jenkins/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1393/test-portal-acceptance-pullrequest(master)/stop.properties'
[EnvInject] - Variables injected successfully.
[description-setter] Description set: <strong>brianchandotcom</strong> - <a href="https://github.com/brianchandotcom/liferay-portal/pull/30639">PR#30639</a> - <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1393/jenkins-report.html">Jenkins Report</a>
Finished: FAILURE

```

brianchandotcom commented:
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/d6449056fa...1771c05cef
